go.mod

```diff
@@ -1,19 +1,26 @@
 module github.com/pingcap/parser
 
 require (
+	github.com/cznic/fileutil v0.0.0-20181122101858-4d67cfea8c87 // indirect
```

Comment:
i don't think you need to modify `go.mod` and `go.sum`.

Reply:
👌